en/appbuilder/generate-app/upload-application-to-github.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Indigo Design App Builder - Upload Application to GitHub
+_description: Indigo Design App Builder enables users to choose between downloading their application locally or uploading it to their GitHub repository.
+_keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch UI kits, GitHub
+---
+# Upload Application To Github
+
+> [!NOTE]
+><b>Once ready with you design in Indigo.Design App Builder you can upload it to your private or Github repository. 
```

Comment:
Do you mean to say private or public GitHub repository? Not that GitHub is written in the way I add in this suggestion
```suggestion
><b>Once ready with your design in Indigo.Design App Builder you can upload it to your private or GitHub repository. 
```